docs/product/issues/issue-details/index.mdx

```diff
@@ -88,23 +107,34 @@ You can set your own <PlatformLink to="/enriching-events/breadcrumbs/">breadcrum
 
 If you’ve enabled [Session Replay](/product/session-replay/), you’ll see a replay preview under Breadcrumbs if there’s one associated with the event you’re viewing. Replays can be associated with both frontend and [backend errors](/product/session-replay/getting-started#replays-for-backend-errors) (as long as distrubted tracing is set up). Clicking on the replay preview will lead you to the [Replay Details](/product/session-replay/replay-details/) page.
 
-## Trace Details
+## Tags
 
-![Trace Details](./img/trace-details.png)
+<Include name="common-imgs/tags" />
 
-[Tracing](/product/sentry-basics/tracing/distributed-tracing/) augments your existing error data by capturing interactions between your software systems. In the "Trace Details" section, you can click the "Search by Trace" button to see all the events that share the same trace ID.
+[Tags](/product/sentry-basics/search/) are key/value string pairs that are both indexed and searchable. For example, a tag provides you with information such as the browser, device, or user associated with the event. Tags are the diagnostic information sent by the SDK for the individual event.
 
-## Tabs
+These event tags can be filtered by one of the available categories, and clicking any of the hover dropdowns will provide you with a few more search options. Sentry will display tags containing '`.`' characters as a tree structure to make things a bit more readable, but searching should always use the full tag name (e.g. `organization.slug` instead of `slug`).
```

Comment:
```suggestion
You can use the tag tabs to filter these event tags by category ("All", "Custom", "Application", "Other").  If you hover over a tag and then click "...", you can choose to view more events or issues with that tag value (and sometimes other information associated with that tag, such as the release itself). Sentry will display tags containing '`.`' characters as a tree structure to make things a bit more readable, but searching should always use the full tag name (for example, `organization.slug` instead of `slug`).
```